tests/unit/src/test/scala/tests/BillLspSuite.scala

```diff
@@ -203,19 +211,4 @@ class BillLspSuite extends BaseLspSuite("bill") {
       )
     } yield ()
   }
-
-  test("conflict") {
-    cleanWorkspace()
-    Bill.installWorkspace(workspace.toNIO, "Bill")
-    Bill.installWorkspace(workspace.toNIO, "Bob")
-    testSelectServerDialogue()
-  }
-
-  test("mix") {
-    cleanWorkspace()
-    RecursivelyDelete(globalBsp)
-    Bill.installWorkspace(workspace.toNIO, "Bill")
-    Bill.installGlobal(globalBsp.toNIO, "Bob")
-    testSelectServerDialogue()
-  }
```

Comment:
I removed these because again, this will never actually happen anymore. In this situation, it would instead just default to Bloop.